ts/features/it-wallet/screens/ItwHomeScreen.tsx

```diff
@@ -23,6 +29,7 @@ const ItwHomeScreen = () => {
   const navigation = useNavigation();
   const isWalletActive = useIOSelector(ItwWalletActivatedSelector);
   const [selectedBadgeIdx, setSelectedBadgeIdx] = useState(0);
+  const pid = useIOSelector(ItwWalletVcsSelector)[0].verified_claims.claims;
```

Comment:
What happen if verified_claims was null?

Reply:
At the moment we are 100% sure that won't be null or undefined. Do we want to handle this specific case right now and maybe show an error? e

Reply:
I have tested on physical device and ItwHome crash for undefined verified_claims.

Reply:
I will try to investigate further

Reply:
That's true, this definitely needs to be fixed. I was confusing the home screen with the details screen. I'll look into it asap.